### DISCUSSION TITLE
Dịch Least Absolute Shrinkage and Selection Operator

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#19] [Terminology] [answered]
Hồi Quy Lasso dịch ra tiếng Việt như thế nào mọi người nhỉ?

* Hồi quy toán tử lựa chọn và co ngót tuyệt đối ít nhất
* Hồi quy toán tử lựa chọn và co tuyệt đối ít nhất
* ...

@mlbvn/handson-ml2

**Comment by lkhphuc** [comment]
Mình thấy giữ nguyên `hồi quy Lasso` cho đơn giản nhỉ, dịch lần đầu thì giải thích thôi.

**Reply by hnguyentt** [reply]
Vì trong sách họ ghi tên đầy đủ ấy @lkhphuc

**Reply by hnguyentt** [reply]
Chắc ai đó rename cái title của mình. Mình ghi tên đầy đủ. 😀

**Comment by minhduc0711** [comment, marked as answer]
em thấy option 2 có vẻ được, nhưng sao không giữ nguyên thứ tự nhỉ
`Hồi quy toán tử co tuyệt đối nhỏ nhất và lựa chọn`

**Reply by tiepvupsu** [reply]
Ui, cụm này dịch một lần rồi dùng Lasso thôi. Cụm này để nguyên tiếng Anh cũng ít người hiểu nhưng Lasso thì nhiều.

**Reply by minhduc0711** [reply]
à chẳng qua là có câu này ạ, nên em mới nghĩ dịch lần đầu cũng được vì nó cũng chỉ xuất hiện 1 lần
```
Least Absolute Shrinkage and Selection Operator Regression (usually simply called Lasso Regression) is another regularized version of Linear Regression:
```

**Reply by minhduc0711** [reply]
từ này dài mà cũng chỉ dùng 1 lần nên chắc khỏi cho vào glossary luôn nhỉ
